dvc/command/diff.py

```diff
@@ -167,6 +170,15 @@ def add_parser(subparsers, parent_parser):
         help=DIFF_DESCRIPTION,
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
+    diff_parser.add_argument(
+        "--targets",
+        nargs="*",
+        help=(
+            "Source paths to a data files or directories. Default None. "
+            "If not specified, compares all files and directories `). "
+            "that are under DVC control in the current working space."
+        ),
```

Comment:
How about "Limit command scope to these tracked files or directories. Accepts one or more file paths."